src/cipher.rs

```diff
@@ -100,6 +100,18 @@ pub extern "C" fn rustls_supported_ciphersuite_get_name(
     }
 }
 
+/// Returns the IANA registered protocol version identifier of the ciphersuite.
+///
+/// See also `RUSTLS_ALL_VERSIONS`.
+#[no_mangle]
+pub extern "C" fn rustls_supported_ciphersuite_protocol(
```

Comment:
I would be minded to have `version` somewhere in the name of this.

Reply:
Renamed to `rustls_supported_ciphersuite_protocol_version()` (my kingdom for namespaces :weary: )